tv_apps/smart-home/test/unit/home_test.js

```diff
@@ -118,6 +118,16 @@ suite('home', function() {
 
     fakeTimer = this.sinon.useFakeTimers();
     window.MozActivity = MockMozActivity;
+
+    if (!Intl.DateTimeFormat.prototype.formatToParts) {
```

Comment:
`Intl.DateTime.prototype.formatToParts` is currently only exposed to B2G certified apps (because we're still in the process of standardizing it in ECMA 402), so it's not available in unit tests.